Rough draft slides Week 1

joeyhuaa opened the pull request.

joeyhuaa committed:
simpler mdx but works smoothly

joeyhuaa committed:
commit

joeyhuaa committed:
commit

joeyhuaa committed:
added test component

joeyhuaa committed:
made 9 slides

joeyhuaa committed:
made 9 empty slides

JeffTheAggie committed:
Update deck.mdx

JeffTheAggie committed:
Update deck.mdx

JeffTheAggie committed:
Update deck.mdx

JeffTheAggie committed:
Update deck.mdx

JeffTheAggie committed:
Update deck.mdx

JeffTheAggie committed:
Update deck.mdx

JeffTheAggie committed:
Update deck.mdx

JeffTheAggie committed:
Update deck.mdx

JeffTheAggie committed:
Update deck.mdx

JeffTheAggie committed:
Update deck.mdx

joeyhuaa committed:
images joey

joeyhuaa committed:
Merge branch 'joey' into Jeffrey

JeffTheAggie committed:
Update deck.mdx

JeffTheAggie committed:
Update deck.mdx

JeffTheAggie committed:
Update deck.mdx

JeffTheAggie committed:
Update deck.mdx

JeffTheAggie committed:
Update deck.mdx

JeffTheAggie committed:
Update deck.mdx

JeffTheAggie committed:
Update deck.mdx

JeffTheAggie committed:
Update deck.mdx

JeffTheAggie committed:
Update deck.mdx

JeffTheAggie committed:
Update deck.mdx

JeffTheAggie committed:
Update deck.mdx

JeffTheAggie committed:
Update deck.mdx

JeffTheAggie committed:
Update deck.mdx

JeffTheAggie committed:
Update deck.mdx

JeffTheAggie committed:
Update deck.mdx

JeffTheAggie committed:
Update deck.mdx

joeyhuaa committed:
Merge pull request #1 from tkashif/Jeffrey

Jeffrey merge --> Joey

JeffTheAggie committed:
Update deck.mdx

joeyhuaa committed:
Merge branch 'Jeffrey' of https://github.com/tkashif/covid-mdx-updated into Jeffrey

joeyhuaa committed:
joey + jeffrey rough

joeyhuaa committed:
Merge branch 'joey' of https://github.com/tkashif/covid-mdx-updated into joey

tkashif commented:
The following is feedback that I have regarding the presentation uploaded to the personal repo.

This is the same feedback that I have commented on the Week 1 of 3 Issue assigned to you guys in the Bit Project MDX repo. On that one, please check off the boxes as you complete the work. After you have completed the work, keep pushing to this "Joey" branch on this personal repo and let me know when everything has been implemented.

Just to reiterate, work and push to the personal repo, but check boxes off on the Bit Project repo. Sorry for the inconvenience.

- A lot of text and images were being cut off when I ran the presentation on the browser. Please fix this ASAP (number one priority) as your hard work isn't being displayed! 
- There are some typos in the presentation, so make sure to read through and ensure those are fixed
- Add speaker notes to the mdx file (see https://diycode.cc/projects/jxnblk/mdx-deck on how to do this). A lot of the slides have a lot of text. The explanations are well thought out, but each bullet point should be summarized succinctly, and the rest of the explanation should be transferred to speaker notes
- Add a transition slide, letting the audience know that they will be coding/exploring the interactive activity on their own. Right now, it kind of seems just like additional information you are going to present, but the focus should be on making sure the audience knows they will be doing the work!
- For the last slide, also include below code somewhere to let audience know that the title of the time series is coming from this. This code is located in the "How to Visualize Postman" tutorial, under the "Add ISO variable" section, under the "Tests" section. 
```
// Set visualizer
pm.visualizer.set(template, {
    // Pass the response body parsed as JSON as `data`
    response: pm.response.json(),
    title: `COVID timeseries data for ${pm.variables.get("ISO")}`
});
```

This feedback should be implemented by Sunday April 12th, so that I can have Kevin review the updated work

JeffTheAggie committed:
Update deck.mdx

Reformatted the content into speaker notes

JeffTheAggie committed:
Update deck.mdx

JeffTheAggie committed:
Update deck.mdx

JeffTheAggie committed:
Update deck.mdx

JeffTheAggie committed:
Update deck.mdx

JeffTheAggie committed:
Update deck.mdx

JeffTheAggie committed:
Update deck.mdx

JeffTheAggie committed:
Update deck.mdx

JeffTheAggie committed:
Update deck.mdx

Added the interactive component, edited the slides for grammatical issues, and made the content notes into speaker notes instead.

JeffTheAggie committed:
Update deck.mdx

JeffTheAggie committed:
Update deck.mdx

JeffTheAggie committed:
Update deck.mdx

JeffTheAggie committed:
Add files via upload

JeffTheAggie committed:
Add files via upload

JeffTheAggie committed:
Add files via upload

JeffTheAggie committed:
Add files via upload

JeffTheAggie committed:
Add files via upload

JeffTheAggie committed:
Add files via upload

JeffTheAggie committed:
Add files via upload

JeffTheAggie committed:
Add files via upload

JeffTheAggie committed:
Add files via upload

JeffTheAggie committed:
Add files via upload

JeffTheAggie committed:
Delete ISO Code.PNG

JeffTheAggie committed:
Delete image.png

JeffTheAggie committed:
Add files via upload

JeffTheAggie committed:
Add files via upload

JeffTheAggie committed:
Delete hrmp3wjiruq9y68i.png

joeyhuaa committed:
made prettier

JeffTheAggie committed:
Add files via upload

joeyhuaa committed:
made prettier

JeffTheAggie committed:
Add files via upload

JeffTheAggie committed:
Delete Backend Code.PNG

JeffTheAggie committed:
Add files via upload

JeffTheAggie committed:
Delete Data Graph.PNG

joeyhuaa committed:
commit

JeffTheAggie committed:
Add files via upload

JeffTheAggie committed:
Add files via upload

joeyhuaa committed:
finished draft

JeffTheAggie committed:
Delete global.PNG

JeffTheAggie committed:
Add files via upload

JeffTheAggie committed:
Add files via upload

JeffTheAggie committed:
Add files via upload

JeffTheAggie committed:
Update deck.mdx

I fixed the syntax for the images on my slides.

joeyhuaa committed:
Merge pull request #7 from tkashif/joey

joey made slides prettier

joeyhuaa committed:
added speaker notes

JeffTheAggie committed:
Delete ISO Graph.PNG

JeffTheAggie committed:
Add files via upload

JeffTheAggie committed:
Update deck.mdx

Made the speaker notes only face the presenter and fixed the ISO graph picture

JeffTheAggie committed:
Delete OSI.png

JeffTheAggie committed:
Add files via upload

JeffTheAggie committed:
Update deck.mdx

Changed the image name

joeyhuaa committed:
merged Jeffrey3

joeyhuaa committed:
merged joey

taimurkashif committed:
changed notes

taimurkashif committed:
reverted notes

joeyhuaa committed:
this is working

joeyhuaa committed:
this is working

JeffTheAggie committed:
Update deck.mdx

Reformatted the speaker notes into paragraph tabs.

JeffTheAggie committed:
Update deck.mdx

Added the </p> to close each paragraph

joeyhuaa committed:
works now

joeyhuaa committed:
merged Jeffrey3

joeyhuaa committed:
merged joey-jeffrey
